resources/js/components/Calendar.vue

```diff
@@ -127,11 +127,10 @@
             parentDiv.classList.add('bg-blue-500');
             }
         },
-        setCurrentDay(day){
+        setCurrentDay(day) {
```

Comment:
qui secondo me va fatto l'emit (che è anche il punto in cui c ambia il giorno, e di conseguenza viene ricalcolato dayMonthAndYear)